forum_service.py

```diff
@@ -1,40 +1,51 @@
 from rs_scraper import RSScraper
+from post_cleaner import PostCleaner
+from datetime import datetime as dt
 
 class ForumService:
     """
-        Handles the scraper.
+        Handles the scraping calls and cleaning data
     """
-    def __init__(self):
-        pass
-
-    ##################
-    # Scraping Methods
-    ##################
-    def get_raw_posts(self, url, page_start, page_end=''):
-        scraper = RSScraper(url)
-        if (page_end == '') : page_end = scraper.max_page #defaults to last page if a specific page is not specified
-        raw_posts = []
-        for i in range(page_start, page_end):
-            raw_posts += scraper.scrape_page(i)
-        return raw_posts, scraper.current_page, scraper.current_post
-        
-
-
 
-    def get_all_price_posts(self, url):
+    def get_clean_forum_posts(self, url, thread_id, page_start, page_end=''):
+        """
+            Takes a url, thread_id, page_start, and optionally, a page_end
+            it will return a list of tuples containing properly formatted forum post information
+            for the specified page range
+        """
+        #defaults to last page if a specific page is not specified
+        if (page_end == '') : page_end = RSScraper(url).get_max_page() + 1 #NOTE: is this bad practice??
+        result = []
+        for page_index in range(page_start, page_end):
+            result += (self.get_clean_forum_post(url, thread_id, page_index))
+        return result
+
+    def get_clean_forum_post(self, url, thread_id, page_num):
```

Comment:
same as above, I think the "clean" is unnecessary

```suggestion
    def get_forum_post(self, url, thread_id, page_num):
```

Reply:
also, I think the name is kind of confusing because it returns a whole page worth of forum posts, not just one